e2e/skyux-lib-help-tests/skyux-lib-help.e2e-spec.js

```diff
@@ -102,11 +83,12 @@ function addModalToHomePage() {
 
 describe('skyux lib help', () => {
   beforeAll((done) => {
-    prepareBuild()
-    .then(() => {
-      done();
-    });
-    addModalToHomePage();
+    common.exec('npm', ['i', '--no-save', '@skyux/modals'])
```

Comment:
Are builder's `devDependencies` not recognized for this? https://github.com/blackbaud/skyux-sdk-builder/blob/master/package.json#L141

Reply:
Ah good catch.  This is an artifact of me debugging the failure.  Reverting now.